### PR TITLE
DEVX-1399 makes ES and Kibana check functions verify by port only

### PR DESCRIPTION
**Pull request by rspurgeon**
Also updated the es script to only invoke curl once, check for curls existence
and cleaned up a little of the conditional logic

### CODE REVIEW

**Review by ybyzek — APPROVED** [review]
LGTM, assuming you have validated for both tarball and Docker?

**Comment by rspurgeon** [comment]
> LGTM, assuming you have validated for both tarball and Docker?

I tested w/ bash only.   The scripts don't run anywhere but locally in a bash shell, and also just check network ports, so there _shouldn't_ be any difference.  But I can hold off on merging and verify with a few of the demos.

**Comment by ybyzek** [comment]
> The scripts don't run anywhere but locally in a bash shell

Good catch, it's for tarball only.  Ship it!

**Pull request merged**
